core/trino-spi/src/main/java/io/trino/spi/eventlistener/EventListener.java

```diff
@@ -34,4 +34,12 @@ default boolean requiresAnonymizedPlan()
     {
         return false;
     }
+
+    default void shutdown()
+            throws Exception
+    {
+        if (this instanceof AutoCloseable autoCloseable) {
```

Comment:
AutoCloseable is not needed. Event Listener isn't used in try-with-resources block anyway